relay-common/Cargo.toml

```diff
@@ -22,7 +22,7 @@ lru = "0.4.0"
 parking_lot = "0.10.0"
 regex = "1.3.9"
 sentry-types = "0.14.1"
-schemars = { version = "0.7.1", features = ["uuid", "chrono"], optional = true }
+schemars = { version = "0.8.0-alpha-4", git = "https://github.com/untitaker/schemars", branch = "preserve-formatting", features = ["uuid", "chrono"], optional = true }
```

Comment:
I'm wondering if it would make sense to re-export `schemars` as a module of `relay-common`, just like we do with `uuid` and some others. Given that we need it to be absolutely compatible across the crates, I think that would make sense.

Reply:
Yeah was wondering how to do that. One problem is `schemars_derive` which I am not sure how to reexport.

Reply:
It *does* fail to compile very spectacularly if there's a version split though, so that's good.